packages/react-reconciler/src/ReactFiberWorkLoop.old.js

```diff
@@ -727,7 +726,10 @@ function ensureRootIsScheduled(root: FiberRoot, currentTime: number) {
 
   // Schedule a new callback.
   let newCallbackNode;
-  if (newCallbackPriority === SyncLanePriority) {
+  if (
+    newCallbackPriority === SyncLanePriority ||
+    newCallbackPriority === InputDiscreteLanePriority
```

Comment:
Instead of this, I think you should change `findUpdateLane` to return `SyncLane`:

https://github.com/facebook/react/blob/60182d64ca030ef28782c7e67be373eac81ae4c2/packages/react-reconciler/src/ReactFiberLane.new.js#L568-L569

Then we're one step closer to deleting the priority level entirely.

Reply:
Done!